### ISSUE TITLE
numpy.core.multiarray failed to import

### ISSUE THREAD

**Issue by penetest** [bytedeco/javacpp-presets#712]
Hi,when  using numpy-platform,show some error:

numpy.core.multiarray failed to import

**Comment by saudet** [comment]
CPython currently isn't getting bundled yet. This is going to get fixed sooner or later, but for now make sure you have CPython 3.6 installed on your system.

**Comment by penetest** [comment]
> CPython currently isn't getting bundled yet. This is going to get fixed sooner or later, but for now make sure you have CPython 3.6 installed on your system.

Hi, saudet.
     Firstly, thank you for your help. My system has installed CPython 3.6 and numpy1.16.2, next what can i do?

**Comment by saudet** [comment]
What is your operating system and how did you install CPython 3.6?

**Comment by penetest** [comment]
> What is your operating system and how did you install CPython 3.6?

sys:  CentOS7.6,
python installed :
     wget https://www.python.org/ftp/python/3.6.8/Python-3.6.8.tgz
     tar -zxvf Python-3.6.8.tgz
     mkdir -p /usr/local/python3
     cd Python-3.6.8
    ./configure --prefix=/usr/local/python3
    make && make install
     ln -s /usr/local/python3/bin/python3 /usr/bin/python3

**Comment by saudet** [comment]
Try to use the official build instead by calling `yum install python36`

**Comment by penetest** [comment]
Thank you for your help! I have a success by the docker image( python:3.6.8),and do some settings in .bashrc ,as fellows:  
```
export PYTHONHOME=/usr/local/bin
export PYTHONPATH=/usr/local/lib/python3.6:/usr/local/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/site-packages

```
**if you not do it, may show some errors:**

**Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'
Current thread 0x00007f3ffcd25700 (most recent call first):
Aborted (core dumped)**

**Comment by saudet** [comment]
CPython and NumPy are now bundled! Please give it a try with the snapshots: http://bytedeco.org/builds/

**Comment by saudet** [comment]
And they have now been released with version 1.5.1. Enjoy!

**Comment by penetest** [comment]
Yes, 1.5.1 is very better，even  not install python
